tests/filterNew.test.js

```diff
@@ -0,0 +1,61 @@
+import {subDays, subMonths, subWeeks} from "date-fns";
+import {filterNew} from "../src/lib";
+
+describe('filterNew()', () => {
+
+  const date1 = subMonths(new Date(), 3).toISOString()
+  const date2 = subMonths(new Date(), 1).toISOString()
+  const date3 = subWeeks(new Date(), 2).toISOString()
+  const date4 = subDays(new Date(), 2).toISOString()
+  const date5 = subDays(new Date(), 1).toISOString()
+  const todayDate = new Date()
+
+
+  it('returns filtered projects that are new based on subDays', () => {
+    const projects = [
+      { created: date3 },
+      { created: date4 },
+      { created: date5 },
+    ]
+
+    process.env.subDays = '3'
+    const filteredProjects = filterNew(projects, todayDate)
+    const expectedProjects = [
+      { created: date4 },
+      { created: date5 },
+    ]
+
+    expect(filteredProjects).toEqual(expectedProjects)
+  })
+
+  it('returns no projects when none are new based on subDays', () => {
+    const projects = [
+      { created: date3 },
+      { created: date4 },
+      { created: date5 },
+    ]
+
+    process.env.subDays = '1'
```

Comment:
I think using function arguments here might be easier for people to understand and easier to maintain – something to think about for a future PR.